integration-tests/src/test/java/oracle/kubernetes/operator/ITUsabilityOperatorHelmChart.java

```diff
@@ -596,4 +581,130 @@ public void testCreateWithDefaultTargetDomainInstall() throws Exception {
+    } catch (Exception ex) {
+      if (!ex.getMessage()
+          .contains("Response {\"status\":404,\"detail\":\"/operator/latest/domains/test" + number))
+        throw new RuntimeException(
+            "FAILURE: Exception does not report the expected error message " + ex.getMessage());
+
+      testCompletedSuccessfully = true;
+    } finally {
+      if (domain != null && !SMOKETEST && (JENKINS || testCompletedSuccessfully))
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+      if (domainnew != null && !SMOKETEST && (JENKINS || testCompletedSuccessfully))
+        domainnew.destroy();
+
+      if (operator != null) {
+        operator.destroy();
+      }
+      number++;
+    }
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+  /**
+   * Create operator and verify its deployed successfully. Create domain1 and verify domain is
+   * started. Delete operator and make sure domain1 is still functional
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testDeleteOperatorButNotDomain() throws Exception {
+    Assume.assumeFalse(QUICKTEST);
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    // create operator
+    Operator operator = null;
+    Domain domain = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> operatorMap = TestUtils.createOperatorMap(number, true);
+      operator = new Operator(operatorMap, RESTCertType.SELF_SIGNED);
+      operator.callHelmInstall();
+      domain = createVerifyDomain(number, operator);
+      logger.info("Deleting operator to check that domain functionality is not effected");
+      operator.destroy();
+      operator = null;
+      Thread.sleep(60 * 1000);
+      domain.testWlsLivenessProbe();
+      testCompletedSuccessfully = true;
+    } finally {
+      if (domain != null && !SMOKETEST && (JENKINS || testCompletedSuccessfully))
+        TestUtils.deleteWeblogicDomainResources(domain.getDomainUid());
+
+      if (operator != null) {
+        operator.destroy();
+      }
+      number++;
+    }
+
+    logger.info("SUCCESS - " + testMethodName);
+  }
+
+  private Domain createVerifyDomain(int number, Operator operator) throws Exception {
+    logger.info("create domain with UID : test" + number);
+    Domain domain = TestUtils.createDomain(TestUtils.createDomainMap(number));
+    domain.verifyDomainCreated();
+    testAdminT3Channel(domain);
+    TestUtils.renewK8sClusterLease(getProjectRoot(), getLeaseId());
+    logger.info("verify that domain is managed by operator");
+    operator.verifyDomainExists(domain.getDomainUid());
+    return domain;
+  }
+
+  private void upgradeVerifyOperatorDomainNamespaces(
+      Operator operator, ArrayList<String> targetNamespaces) throws Exception {
```

Comment:
indentation

Reply:
mvn format (mvn com.coveo:fmt-maven-plugin:format)always put it in this format
 private void upgradeOperatorDomainNamespaces(
      Operator operator, ArrayList<String> targetNamespaces) throws Exception {
not sure what to do about it